controller/cluster/logging_test.go

```diff
@@ -57,40 +58,82 @@ var _ = Describe("Logging", func() {
 	})
 
 	When("cluster is added", func() {
-		It("event should be logged", func(done Done) {
-			var event controller.Event
-
-			ctx, cancelFunc := context.WithCancel(context.Background())
-			go clusterController.Run(ctx)
-			defer cancelFunc()
-
-			// Wait for
-			Eventually(eventsHook.GetEventsChan()).Should(Receive(&event))
-			Expect(logrusHook.AllEntries()).To(HaveLen(2))
-
-			By("outputting on event received")
-			firstEntry := logrusHook.AllEntries()[0]
-			Expect(*firstEntry).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
-				"Level":   Equal(logrus.InfoLevel),
-				"Message": Equal("Received cluster event"),
-				"Data": Equal(logrus.Fields{
-					"event":   ClusterAdded,
-					"cluster": cluster.Name,
-				}),
-			}))
-
-			By("outputting on event processed")
-			secondEntry := logrusHook.AllEntries()[1]
-			Expect(*secondEntry).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
-				"Level":   Equal(logrus.InfoLevel),
-				"Message": Equal("Successfully processed cluster event"),
-				"Data": Equal(logrus.Fields{
-					"event":   ClusterAdded,
-					"cluster": cluster.Name,
-				}),
-			}))
+		Describe("and succesfully created", func() {
+			It("event should be logged", func(done Done) {
+				var event controller.Event
+
+				ctx, cancelFunc := context.WithCancel(context.Background())
+				go clusterController.Run(ctx)
+				defer cancelFunc()
+
+				// Wait for
+				Eventually(eventsHook.GetEventsChan()).Should(Receive(&event))
+				Expect(logrusHook.AllEntries()).To(HaveLen(2))
+
+				By("outputting on event received")
+				firstEntry := logrusHook.AllEntries()[0]
+				Expect(*firstEntry).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
+					"Level":   Equal(logrus.InfoLevel),
+					"Message": Equal("Received cluster event"),
+					"Data": Equal(logrus.Fields{
+						"event":   ClusterAdded,
+						"cluster": cluster.Name,
+					}),
+				}))
+
+				By("outputting on event processed")
+				secondEntry := logrusHook.AllEntries()[1]
+				Expect(*secondEntry).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
```

Comment:
Maybe write a wrapper for the whole `gstruct...` match as it's often used throughout the tests?